source/extensions/tracers/skywalking/trace_segment_reporter.cc

```diff
@@ -0,0 +1,177 @@
+#include "extensions/tracers/skywalking/trace_segment_reporter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Tracers {
+namespace SkyWalking {
+
+namespace {
+
+const Http::LowerCaseString& authenticationTokenKey() {
+  CONSTRUCT_ON_FIRST_USE(Http::LowerCaseString, "Authentication");
```

Comment:
You should do the Inline header registration, see usage of `Http::RegisterCustomInlineHeader<Http::CustomInlineHeaderRegistry::Type::RequestHeaders>`

Reply:
Then, we need add `authentication` to CustomHeaderValues.

Reply:
Is there a clear criterion for which headers should be added to CustomHeadersValue and which should not?

Reply:
ha. I find #4815 and #11546.